assets/src/scripts/components/hydrograph/scales.js

```diff
@@ -18,12 +18,13 @@ const { getVariables, getCurrentParmCd, getRequestTimeRange } = require('../../s
  * @return {Object} d3 scale for time.
  */
 function createXScale(timeRange, xSize) {
-    const xExtent = timeRange ? [timeRange.start, timeRange.end] : [0, 1];
-
     // xScale is oriented on the left
-    return scaleTime()
-        .range([0, xSize])
-        .domain(xExtent);
+    let scale = scaleTime()
+        .range([0, xSize]);
+    if (timeRange) {
+        scale.domain([timeRange.start, timeRange.end]);
+    }
```

Comment:
Curious - what does `scale.domain()` return here, if `timeRange` isn't set? I thought `[0, 1]` is always the default with D3. Must be reading the code wrong... but scaleTime is a continuous scale: https://github.com/d3/d3-scale/blob/master/src/continuous.js#L7

Reply:
default is [2000-01-02, 200-01-02]. see https://github.com/d3/d3-scale/blob/master/README.md#scaleTime